project/build.properties

```diff
@@ -1,2 +1,2 @@
-sbt.version=1.6.2
+sbt.version=1.9.7
```

Comment:
I would probably keep the old version here - this is also the version we build the plugin against, and I think it's generally not guaranteed that a plugin will work with older sbt.

Reply:
hmm, afaik sbt version only affects build process not the the resulting artifact, but having said that, i never maintained sbt plugin...
it's not necessary to do it right now now, but it's good practice to keep it up to date, and all 1.x series should be binary compatible, so how would you go about updating sbt version?

Reply:
https://www.scala-sbt.org/1.x/docs/Plugins.html#Creating+an+auto+plugin

> By default sbt plugin is compiled with whichever the sbt version you are using. Because sbt does NOT keep forward compatibility, that would typically require all of your plugin users to upgrade to the latest too. pluginCrossBuild / sbtVersion is an optional setting to compile your plugin against an older version of sbt, which allows the plugin users to choose from a range of sbt versions.

We can update sbt used by the project, but then it makes sense to specify `pluginCrossBuild / sbtVersion` to use some old version.

Reply:
nice, i added it to the project with version 1.6.2 which was previous version